### ISSUE TITLE
attempt to call field 'win_open' (a nil value)

### ISSUE THREAD

**Issue by GustavoPrietoP** [kyazdani42/nvim-tree.lua#289]
I don't know if this has to do with a recent update or not but I recently updated my nvim-tree and its giving me this error
![nvim-tree](https://user-images.githubusercontent.com/70907734/114632608-0bf72480-9c8d-11eb-9f55-a9c58b1db08d.png)

**Comment by agibson-bluecat** [comment]
Looks like this function was moved to view.lua, and updated slightly (used to be in lib.lua). Changing the call at nvim-tree.lua line 165 from lib.win_open() to view.win_open() does the trick.

**Comment by agibson-bluecat** [comment]
Just a slight oversight from the most recent commit (ff814676), which was made today

**Comment by yashguptaz** [comment]
I am experiencing the same issue after the update.

**Comment by ttys3** [comment]
```diff

diff --git a/lua/nvim-tree.lua b/lua/nvim-tree.lua
index 0eb4e54..633c77f 100644
--- a/lua/nvim-tree.lua
+++ b/lua/nvim-tree.lua
@@ -162,7 +162,7 @@ end

 function M.on_leave()
   vim.defer_fn(function()
-    if #api.nvim_list_wins() == 1 and lib.win_open() then
+    if #api.nvim_list_wins() == 1 and view.win_open() then
       api.nvim_command(':silent qa!')
     end
   end, 50)
diff --git a/lua/nvim-tree/lib.lua b/lua/nvim-tree/lib.lua
index 6958462..fcf7513 100644
--- a/lua/nvim-tree/lib.lua
+++ b/lua/nvim-tree/lib.lua
@@ -245,7 +245,7 @@ function M.open_file(mode, filename)
   view.resize()

   if vim.g.nvim_tree_quit_on_open == 1 and mode ~= 'preview' then
-    M.close()
+    view.close()
   end

   renderer.draw(M.Tree, true)
```

**Comment by yashguptaz** [comment]
It seems like nvim-tree is using a new Neovim api.
Updating Neovim nightly build seems to fix the issue.

**Comment by kyazdani42** [comment]
sorry, forgot to change this func call. It's fixed in d065927
